<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>574</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>321</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="select_converter">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Select Converter</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="select_from">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>From</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_from"/>
         </item>
         <item>
          <widget class="QComboBox" name="select_to">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>To</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_convert">
           <property name="text">
            <string>Convert</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="display_results"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
